targets/mbedos5/.gitignore

```diff
@@ -0,0 +1,8 @@
+mbed-os
+mbed-events
+.build
+.mbed
+mbed_settings.py
```

Comment:
mbed_settings.py is created by mbed cli in mbedos5, but is specific to individual users' machines, so shouldn't be under version control.